README.md

```diff
@@ -1,4 +1,91 @@
-# team-awesome-amijs
+# ✨✨✨ TEAM 💫 AWESOME 💫 AMIJS ✨✨✨
 
-ayeee
-ayooo
+A _majestic_ hour registration app.
+
+## ☄️ The purpose of this app
+
+This app is used for registering working hours. As Kanye West once said:
+
+> Work it, make it, do it
+
+## 🐉 The technologies being used
+
+- [Cypress](https://www.cypress.io/)
+- [Jest](https://jestjs.io/)
+- [Next.js](https://nextjs.org/)
+- [Storybook](https://storybook.js.org/)
+- [Zustand](https://github.com/pmndrs/zustand)
+
+## 🌈 How to run this application
+
+To run Team Awesome, checkout the repository and use
+
+```
+npm run dev
+```
+
+To run tests using Cypress and Jest, use either
```

Comment:
This implies you will run both kinds by running either of the following commands